app/portainer/settings/authentication/ldap/ldap-group-search/ldap-group-search.html

```diff
@@ -13,15 +13,9 @@
 
 <div class="form-group" style="margin-top: 10px">
   <div class="col-sm-12">
-    <button
-      class="label label-default interactive vertical-center"
-      style="border: 0"
-      ng-click="$ctrl.onAddClick()"
-      limited-feature-dir="{{::$ctrl.limitedFeatureId}}"
-      limited-feature-tabindex="-1"
-    >
-      <pr-icon icon="'plus-circle'"></pr-icon>
-      add group search configuration
```

Comment:
same question?

Reply:
The group search section in the AD settings and the open LDAP settings are both BE only and don't need a teaser so disabling is ok